html_app/assignments/sutd_intro_biology_ps2/instructions.soy

```diff
@@ -0,0 +1,16 @@
+{namespace scb_assignment_sutd_intro_biology_ps2}
+
+/**
+
+*/
+{template .experiment_setup}
+<ul role='list'  class='scb_assignment_specific_tufts_experiment_setup'>
```

Comment:
Is it an issue that the class here appears to be from a different assignment? 'scb_assignment_specific_tufts_experiment_setup' ?